lib/segment/src/index/field_index/full_text_index/mutable_inverted_index.rs

```diff
@@ -79,19 +79,29 @@ impl InvertedIndex for MutableInvertedIndex {
         &mut self,
         point_id: PointOffsetType,
         document: Document,
+        hw_counter: &HardwareCounterCell,
     ) -> OperationResult<()> {
         self.points_count += 1;
+
+        let counter = hw_counter.payload_index_io_write_counter();
+
         if self.point_to_docs.len() <= point_id as usize {
-            self.point_to_docs
-                .resize_with(point_id as usize + 1, Default::default);
+            let new_len = point_id as usize + 1;
+
+            counter.incr_delta(new_len - self.point_to_docs.len());
+
+            self.point_to_docs.resize_with(new_len, Default::default);
         }
 
         for token_idx in document.tokens() {
             let token_idx_usize = *token_idx as usize;
+
             if self.postings.len() <= token_idx_usize {
-                self.postings
-                    .resize_with(token_idx_usize + 1, Default::default);
+                let new_len = token_idx_usize + 1;
+                counter.incr_delta(new_len - self.postings.len());
```

Comment:
For these loops we could do the same thing you do in other places where we accumulate the total delta in a local variable and only `.incr_delta` once to avoid accessing the cell too often